README.md

```diff
@@ -167,6 +167,48 @@ let package = Package(
 )
 ```
 
+#### Adding custom configuration
+It is possible to add Accio comments to the `Package.swift` file. They are used for passing to Accio additional configuration that can not be specified using the official manifest format. Example:
+
+```swift
+// swift-tools-version:5.0
+import PackageDescription
+
+let package = Package(
+    name: "XcodeProjectName",
+    products: [],
+    dependencies: [
+        .package(url: "https://github.com/Flinesoft/HandySwift.git", .upToNextMajor(from: "2.8.0")),
+        .package(url: "https://github.com/Flinesoft/HandyUIKit.git", .upToNextMajor(from: "1.9.0")),
+        .package(url: "https://github.com/Flinesoft/Imperio.git", .upToNextMajor(from: "3.0.0")),
+        .package(url: "https://github.com/JamitLabs/MungoHealer.git", .upToNextMajor(from: "0.3.0")),
+        .package(url: "https://github.com/SwiftyBeaver/SwiftyBeaver.git", .upToNextMajor(from: "1.6.2")),
+    ],
+    targets: [
+        .target(
+            name: "AppTargetName",
+            dependencies: [
+              // accio product-type:static-framework
```

Comment:
Seeing this, I clearly prefer something that actually looks like Swift. Something that could potentially be merged into SwiftPM itself. This also goes in line with the idea to pitch any improvements we make on top of SwiftPM directly back to SwiftPM itself so sometime in the future we might simply uncomment the additional configuration code and it would just work as expected (like we already did with the `iOS` type of product which was introduced in Swift 5 but we added support back to Swift 4.2 based projects.

I'm thinking something like this:

```swift
targets: [
    .target(
        name: "...",
        dependencies: [
            "...", "..."
        ],
        // defaultLinking: Linking.static,
        // customLinkings: [Linking.dynamic: ["HandySwift", "HandyUIKit"]],
        path: "..."
]
```

We could discuss the naming and how to separate them, but I hope you get the basic idea of how the additional configuration should still be implemented in a way that could be natural to SwiftPM's current way of configuration, which is basically written in Swift itself. We can then also use SwiftSyntax to read the values out if it gets more complicated, but Regexes should be probably used at the beginning.